src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -372,6 +378,13 @@ void GcsPlacementGroupScheduler::CommitAllBundles(
         // Update the resource in gcs resource manager
         auto &resources = bundle->GetFormattedResources();
         gcs_resource_manager_.UpdateResources(node_id, resources);
+
```

Comment:
I find that this logic is moved outside from `GcsResourceManager::UpdateResources`, This will cause the data to be synchronized to other nodes before the GCS writes the data to redis. I am not sure if it has problem If the GCS dead at this time?

Reply:
I think this is just to broadcast the resource information and ideally, the correctness shouldn't matter here since the cluster is supposed to be eventually consistent on something.
And resource should be reported also in the normal resource reporting path.